plugins/aws/fix_plugin_aws/resource/backup.py

```diff
@@ -121,7 +123,7 @@ def add_tags(backup_job: AwsBackupJob) -> None:
                 "list-tags",
                 "Tags",
                 expected_errors=["ResourceNotFoundException"],
-                ResourceArn=backup_job.arn,
+                ResourceArn=backup_job.recovery_point_arn,
```

Comment:
You would query for tags of the recovery point - that's wrong isn't it?
Does a job even have tags?

Reply:
Tags from the original resource are automatically copied to the recovery points created during backup jobs. This ensures that the backup data retains the tagging information of the original resources.

https://docs.aws.amazon.com/aws-backup/latest/devguide/tags-on-backups.html

But yes, it is not a backup job/copy job/restore job tags